### PR TITLE
Server log option

### PR DESCRIPTION
**Pull request by g-price**
Added the ability to set custom log file path and name in Server.start()

### CODE REVIEW

File: browsermobproxy/server.py [hunk 1]
```diff
@@ -86,12 +86,16 @@ def __init__(self, path='browsermob-proxy', options={}):
...
-    def start(self):
+    def start(self, options={}):
         """
         This will start the browsermob proxy and then wait until it can
         interact with it
         """
-        self.log_file = open(os.path.abspath('server.log'), 'w')
+        log_path = options.get('path', os.getcwd())
+        log_file = options.get('file', 'server.log')
+        log_path_name = log_path + os.path.sep + log_file
```
Comment (nicolasunravel) [inline browsermobproxy/server.py]: os.path.join here ?

File: browsermobproxy/server.py [hunk 2]
```diff
@@ -86,12 +86,16 @@ def __init__(self, path='browsermob-proxy', options={}):
             self.command = []
         self.command += [path, '--port=%s' % self.port]
 
-    def start(self):
+    def start(self, options={}):
```
Comment (nicolasunravel) [inline browsermobproxy/server.py]: pls don't use a mutable default

File: browsermobproxy/server.py [hunk 3]
```diff
@@ -86,12 +86,16 @@ def __init__(self, path='browsermob-proxy', options={}):
...
         """
         This will start the browsermob proxy and then wait until it can
         interact with it
         """
-        self.log_file = open(os.path.abspath('server.log'), 'w')
+        log_path = options.get('path', os.getcwd())
+        log_file = options.get('file', 'server.log')
+        log_path_name = log_path + os.path.sep + log_file
+
+        self.log_file = open(log_path_name, 'w')
```
Comment (nicolasunravel) [inline browsermobproxy/server.py]: also change error message below

**Comment by AutomatedTester** [comment]
Closing after inactivity (originally on my part) and now on this contribution comments.

**Pull request closed without merging**